### PR TITLE
Create basic configuration for vue3

### PR DESCRIPTION
**Pull request by raimund-schluessler**
This is the PR to create the `next` branch with a basic configuration for `vue 3`, into which we will merge every component migrated to vue 3 one by one.

The branch includes only three migrated components yet, `NcButton`, `NcCheckboxRadioSwitch` and `NcLoadingIcon`.

I kept all non-migrated components and files as they were, just disabled jest, lint, stylelint, styleguide and cypress for them. This way, the git changes are minimal, I hope.

~@skjnldsv @nickvergessen or anyone else with the appropriate rights: Could you please protect the `next` branch and enable building the styleguide for it? Building the styleguide is not triggered here yet, but it works fine locally though.~ Thanks @skjnldsv!

### CODE REVIEW

**Comment by skjnldsv** [comment]
I think you're goo, protected and enabled on netlify. Maybe you need to amend to restart the trigger

**Review by skjnldsv — APPROVED** [review]

**Comment by skjnldsv** [comment]
That would be 9.0.0 though :)

**Comment by raimund-schluessler** [comment]
> That would be 9.0.0 though :) image

I bumped the version to 9.0.0-alpha.0.

**Review by ShGKme — APPROVED** [review]
Reviewed, tested, works 🥳💃

Thanks ❤️

File: src/components/NcButton/NcButton.vue [hunk 1]
```diff
@@ -481,14 +476,14 @@ export default {
...
 	 * The render function to display the component
 	 *
-	 * @param {Function} h The function to create VNodes
 	 * @return {object|undefined} The created VNode
 	 */
-	render(h) {
-		const text = this.$slots.default?.[0]?.text?.trim?.()
+	render() {
+		const text = this.$slots?.default?.()?.[0]?.children?.trim?.()
+			|| this.$slots?.default?.()?.[0]?.children?.[0]?.children?.trim?.()
```
[the remark this answers is not in the document]
Comment (raimund-schluessler) [reply src/components/NcButton/NcButton.vue:483]: > UPD: Though, I'd prefer to get rid of using slots as text props

Yes, I would also like that. But I decided to simply migrate the components and not change their behaviour too much for now. But maybe we change that in `master` for `8.0` already, then we could of course back-port to `next`.

**Pull request merged**